python/paddle/fluid/tests/unittests/test_allclose_op.py

```diff
@@ -22,19 +22,20 @@ class TestAllcloseOp(OpTest):
     def set_args(self):
```

Comment:
Add the test case below.
![image](https://user-images.githubusercontent.com/17102274/95840128-5f3bf900-0d76-11eb-8047-87f2f32143a0.png)

Reply:
Done.